src/javascript/events/component-view.js

```diff
@@ -6,7 +6,7 @@ const getTrace = require('../../libs/get-trace');
 const utils = require('../utils');
 
 const TRACKING_ATTRIBUTES = [
-	'contentId',
+	'componentContentId',
```

Comment:
Is this a breaking change?

Reply:
@JakeChampion It is, but this component-view hasn't been used anywhere yet...